indigo.ignored

```diff
@@ -1,3 +1,4 @@
 chomp_interface
```

Comment:
chomp_interface should be removed since it was renamed to moveit_planners_chomp here https://github.com/ros-planning/moveit/pull/306